### ISSUE TITLE
Can it fly without ever using the Crazyradio?

### ISSUE THREAD

**Issue by ricardodeazambuja** [pulp-platform/pulp-dronet#7]
Hi, 

I was reading your paper "A 64mW DNN-based Visual Navigation Engine for Autonomous Nano-Drones" + the readme file from this repo and I got the idea the system presented is fully autonomous.

So, I'm interested in the possibility of using a Crazyflie without ever connecting to it to my PC with the Crazyradio and I would like to know if in the experiments presented by your paper you had to use the Crazyradio or not (there's no mention to it). 

Thanks!

**Comment by DP8** [comment]
Yes, as we wrote in the aforementioned paper, the system is fully autonomous. Therefore, there is no need to have Crazyradio connected to the drone to perform the experiment -- even though it might be useful for logging and emergency commands.

**Comment by ricardodeazambuja** [comment]
Thanks for the reply! Were you using the Flow deck during the experiments?

**Comment by LorenzoLamberti94** [comment]
> @DP8 I am wondering the control structure of CF2. What is running on CF2 for control, like ROS or PX4...

https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/userguides/app_layer/
